tutor/src/screens/student-dashboard/cells.js

```diff
@@ -54,9 +55,12 @@ const ScoreCell = styled(Cell).withConfig({
   }
 `;
 
-const MobileCell = styled.div`
+const MobileRow = styled.div.attrs({
+  as: 'a',
+})`
```

Comment:
heh, you can just do: `styled.a` here instead of making it a div and then switching to `a` 😁